firebase_admin/auth.py

```diff
@@ -84,11 +93,13 @@
 ActionCodeSettings = _user_mgt.ActionCodeSettings
 CertificateFetchError = _token_gen.CertificateFetchError
 DELETE_ATTRIBUTE = _user_mgt.DELETE_ATTRIBUTE
+DeleteUsersResult = _user_mgt.DeleteUsersResult
```

Comment:
Also BatchDeleteErrorInfo

Reply:
Done. (Also BatchDeleteAccountsResponse)

Reply:
`BatchDeleteAccountsResponse` does not belong in the public API if I understood correct. `BatchDeleteErrorInfo` is exposed via `DeleteUsersResult.errors` so only that's needed.